src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/user/mappers/CaptchaMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.sublinks.sublinksapi.api.lemmy.v3.user.mappers;
+
+import com.sublinks.sublinksapi.api.lemmy.v3.user.dto.Captcha;
+import com.sublinks.sublinksapi.api.lemmy.v3.user.models.GetCaptchaResponse;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
+
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
+public interface CaptchaMapper {
+
+  @Mapping(target = "ok.png", source = "png")
+  @Mapping(target = "ok.wav", source = "wav")
+  @Mapping(target = "ok.uuid", source = "uuid")
+  GetCaptchaResponse map(Captcha captcha);
```

Comment:
We normaly map the lowest possible way ( https://join-lemmy.org/api/interfaces/CaptchaResponse.html )

And when he does a getCaptcha request it maps to the CaptchaResponse => Uses then the builder ( GetCaptchaResponse.builder().ok(mappedCaptchaResponse).build() ) and returns the result. With it its rather more usuable for other instances ( because they can work easier with the lowest possible object. )